src/views/HomeView.vue

```diff
@@ -83,6 +86,17 @@ const releaseTransaction = async (e2eId: string) => {
   }
 };
 
+watch(walletAddress, async () => {
+  const walletLocks = await checkUnreleasedLocks(walletAddress.value);
+  if (walletLocks.pixKey != "") {
+    flowStep.value = Step.Buy;
+    tokenAmount.value = walletLocks.value;
+    pixTarget.value = walletLocks.pixKey;
+  } else {
+    flowStep.value = Step.Search;
+  }
+});
+
```

Comment:
It will be necessary to watch network change and call this method again!

Reply:
Also, it's necessary to watch props change on QrCode Component, because, if a wallet is changed, the Pix object generated from unreleased method will be changed too. So, this changes need to take effect.

Reply:
> Also, it's necessary to watch props change on QrCode Component, because, if a wallet is changed, the Pix object generated from unreleased method will be changed too. So, this changes need to take effect.

I've fixed this by adding a key attribute to QRCodeComponent.